SmartDeviceLink/public/SDLKeyboardProperties.h

```diff
@@ -58,11 +68,16 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nullable, strong, nonatomic) NSArray<NSString *> *limitedCharacterList;
 
 /**
- Allows an app to prepopulate the text field with a suggested or completed entry as the user types
-
- Optional
+ * Deprecated, use autoCompleteList instead.
+ * {"string_min_length": 1, "string_max_length": 1000}
+ *
+ * @deprecated in SmartDeviceLink 6.0.0
+ * @added in SmartDeviceLink 3.0.0
  */
-@property (nullable, strong, nonatomic) NSString *autoCompleteText;
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+@property (nullable, strong, nonatomic) NSString *autoCompleteText __deprecated;
```

Comment:
There are several places in the library where the deprecated `autoCompleteText` or deprecated `init` are used. If the new `init` can be used, the code should be updated to use the new init. Otherwise the old init needs to be wrapped with `#pragma clang diagnostic push/pop`. Anywhere that `autoCompleteText` is set should get wrapped with `#pragma clang diagnostic push/pop`. Using `#pragma clang diagnostic push/pop` will suppress the warning messages.

You should be able to see the warnings in the sidebar after building the library (To show the Issue Navigator open **View > Navigators > Issues**)